### ISSUE TITLE
Rule: `no-ambiguity-target`

### ISSUE THREAD

**Issue by graberzz** [effector/eslint-plugin#133]
This code triggers warning:

```js
import { createStore, createEvent, sample } from "effector";

const obj = {
  fn: () => {
    const $store = createStore(0);
    const event = createEvent();
    // warning  Method `sample` returns `target` and assigns the result to a variable. Consider removing one of them  effector/no-ambiguity-target
    sample({
      source: event,
      target: $store,
    });
  },
};

```

**Comment by igorkamyshev** [comment]
Sounds like a bug 😇 thanks for report, I'll take a look
